src/client/firefox/commands.js

```diff
@@ -160,6 +160,10 @@ type EvaluateParam = {
 
 function evaluate(script: Script, { frameId }: EvaluateParam) {
   const params = frameId ? { frameActor: frameId } : {};
+  if (!tabTarget || !tabTarget.activeConsole) {
+    return Promise.resolve();
```

Comment:
Just added an issue for attaching to the console synchronously.

https://bugzilla.mozilla.org/show_bug.cgi?id=1382302